core/display.js

```diff
@@ -380,6 +380,17 @@ export default class Display {
         });
     }
 
+    videoFrame(x, y, width, height, frame) {
+        this._renderQPush({
+            'type': 'frame',
+            'frame': frame,
+            'x': x,
+            'y': y,
+            'width': width,
+            'height': height
+        });
+    }
```

Comment:
I'm not too thrilled about adding stuff to the general Display class that is specific to one decoder.

But I suppose it is small enough that we can refactor it later if needed.